### PR TITLE
Feature/lp 190 partnership type

### PR DESCRIPTION
**Pull request by lduranteau**
### JIRA link
https://companieshouse.atlassian.net/browse/LP-190

### Change description
which-type page

### Work checklist

- [x] Tests added where applicable
- [ ] UI changes look good on mobile
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: locales/cy/translations.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "buttons" : {
+        "continue" : "WELCH - Continue",
```
Comment (markpit) [inline locales/cy/translations.json:3]: Typo in language name WELCH should be WELSH

File: src/presentation/test/integration/registration/partnership-type.test.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
+import request from "supertest";
+
+import app from "../app";
+import { WHICH_TYPE_URL } from "../../../controller/registration/Routing";
+import { appDevDependencies } from "../../../../config/dev-dependencies";
+import enTranslationText from "../../../../../locales/en/translations.json";
+import RegistrationPageType from "presentation/controller/registration/PageType";
```
Comment (markpit) [inline src/presentation/test/integration/registration/partnership-type.test.ts:7]: the import looks like it is missing the relative path ../../../etc. is it ok?

File: locales/cy/translations.json (hunk 1 above)
Comment (lduranteau) [reply locales/cy/translations.json]: typo fixed

File: src/presentation/test/integration/registration/partnership-type.test.ts (hunk 2 above)
Comment (lduranteau) [reply src/presentation/test/integration/registration/partnership-type.test.ts]: import fixed

**Review by markpit — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Pull request merged**